code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -69,22 +69,23 @@ Difficulty: Medium
 	legiontwo.crusher_loot = list(/datum/nothing)
 	legiontwo.health = 1250
 	legiontwo.maxHealth = 1250
+	legiontwo.enraged = TRUE
 
 /mob/living/simple_animal/hostile/megafauna/legion/unrage()
 	. = ..()
 	for(var/mob/living/simple_animal/hostile/megafauna/legion/other in GLOB.mob_list)
 		if(other != src)
-			other.loot = initial(loot)
-			other.crusher_loot = initial(crusher_loot)
+			other.loot = list(/obj/item/storm_staff)
+			other.crusher_loot = list(/obj/item/storm_staff, /obj/item/crusher_trophy/empowered_legion_skull)
```

Comment:
while you're here, can you add that you cant use initial() here because of lists?

Reply:
I used initial before. It did not work

Reply:
can do

Reply:
done